spec/classes/sensu_init_spec.rb

```diff
@@ -402,7 +402,7 @@
           context "when #{var_name} (#{type}) is set to invalid #{invalid} (as #{invalid.class})" do
             let(:params) { [mandatory_params, var[:params], { :"#{var_name}" => invalid, }].reduce(:merge) }
             it 'should fail' do
-              expect { should contain_class(subject) }.to raise_error(Puppet::Error, /#{var[:message]}/)
+              expect { should contain_class(subject) }.to raise_error(Puppet::PreformattedError)
```

Comment:
Oops, looks my my reply here got lost... 
so yes, I should update the message texts and check on them.
Wonder it it's actually really necessary as they are mostly normal Puppet type validation error messages, so not actually under the module code control.